### PR TITLE
Sorts a text file's words based on the number of their occurences

### PR DESCRIPTION
**Pull request by syedalizar**
Kindly review my work and let me know what I could've done better.

### CODE REVIEW

File: task1.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+from collections import Counter
+
+with open("textfile") as f:
+    txtlist = f.read().split()
```
Comment (UsmanAbbasi1) [inline task1.py]: better name for this variable can be 'words'.

File: task1.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+countdict = (Counter(txtlist))
```
Comment (UsmanAbbasi1) [inline task1.py]: when using less famous features of language, make sure to add comment to explain it.

File: task1.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+
+gen = ((k,v) for (k,v) in countdict.iteritems()) #generator use
+sorteddict = sorted(gen, key=lambda x:x[1], reverse = True)
```
Comment (UsmanAbbasi1) [inline task1.py:11]: explain the logic of these 2 lines (i.e. 10,11) with small comment here.

Comment (syedalizar) [reply task1.py:11]: it creates a generator object 'gen' using list comprehension each instance of which is a tuple (key, value) from 'countdict' which is being iterated over via the 'iteritems' method.
Then the 'sorted' method sorts the generator object based on the key which is identified by the lambda x:x[1]. This lambda method returns the value at index 1 'x[1]' for each tuple 'x' in generator list of tuples 'gen'

File: task1.py (hunk 2 above)
Comment (syedalizar) [reply task1.py]: I removed most of the comments from the code before uploading. 
Will make sure the code is properly commented next time.

File: task1.py (hunk 1 above)
Comment (syedalizar) [reply task1.py]: changes made accordingly in the recent commit

Comment (syedalizar) [reply task1.py]: link to commit:
https://github.com/syedalizar/Arbisoft-Internship/commit/c12754b7e73b9f7da13f2a8a29fcc7f6f8bcc462

**Pull request merged**